Repository/AbstractPhpcrRepository.php

```diff
@@ -110,4 +112,12 @@ protected function unresolvePath($path)
      * @return ArrayResourceCollection
      */
     abstract protected function buildCollection(array $nodes);
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getVersions($path)
+    {
+        return new VersionList($path, [new CmfResource($path)]);
```

Comment:
When in doubt don't extend. I don't know if either of the other two
methods are relevant to us. Would prefer to only extend where it makes
absolute sense.


On Thu, May 19, 2016 at 12:52:05PM -0700, Maximilian Berghoff wrote:
>    In [1]Repository/AbstractPhpcrRepository.php:
> 
>  > @@ -110,4 +112,12 @@ protected function unresolvePath($path)
>  >       * @return ArrayResourceCollection
>  >       */
>  >      abstract protected function buildCollection(array $nodes);
>  > +
>  > +    /**
>  > +     * {@inheritdoc}
>  > +     */
>  > +    public function getVersions($path)
>  > +    {
>  > +        return new VersionList($path, [new CmfResource($path)]);
> 
>    ok, will do that ...
>    but why is there an abstact repository with a implementation, that we
>    don't extend instead?
> 
>    —
>    You are receiving this because you were mentioned.
>    Reply to this email directly or [2]view it on GitHub
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/symfony-cmf/resource/pull/15#discussion_r63942472
>    2. https://github.com/symfony-cmf/resource/pull/15/files/318868bc8aae59467545426bf044226df2519641#r63942472
>    3. https://github.com/symfony-cmf/resource/pull/15/files/318868bc8aae59467545426bf044226df2519641#r63942472